Packs/MISP/Integrations/MISPV3/MISPV3.yml

```diff
@@ -13,6 +13,21 @@ configuration:
   hiddenusername: true
   display: ""
   required: false
+- display: Use IDS flag
+  name: check_to_ids
+  defaultvalue: "true"
+  type: 8
+  required: false
+  additionalinfo: This is to enable checking of the boolean flag to_ids. The flag
+    allows you to indicate if an attribute should be actionable or not
+- display: ORG names to use for reputation checks
+  name: allowed_orgs
+  type: 0
+  required: false
+  additionalinfo: Comma-separated list of allowed TI providers (orgc in MISP events). 
+    When pulling data from MISP you may have data coming from a reliable source.
+    However, some of the orgs are including things like 8.8.8.8 in their feeds.
```

Comment:
if this is a comma separated list, you need to add:
IsArray: true

Reply:
@RosenbergYehuda see the changes. I followed the way how it is defined for other parameters

Reply:
You are right, I did not realize that "Malicious tag IDs" and "Suspicious tag IDs" are type 12.